stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/ChatComponentFactory.kt

```diff
@@ -1178,6 +1186,47 @@ public interface ChatComponentFactory {
         DefaultMessageItemTrailingContent(messageItem = messageItem)
     }
 
+    /**
+     * The default list of reactions displayed above the message bubble and is part of [MessageItemHeaderContent].
+     */
+    @Composable
+    public fun MessageReactionList(
+        modifier: Modifier,
+        message: Message,
+        reactions: List<ReactionOptionItemState>,
+        onClick: (message: Message) -> Unit,
```

Comment:
Can we change it by a "holder object"?
It will help us to include extra argument in the future if it is needed